### DISCUSSION TITLE
Point to `stabilityDays` docs instead of homepage for our `renovate/stabilitydays` CI check

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#14948] [Suggest an Idea]
## Introduction

Renovate bot can add a CI status that checks if the set number of `stabilityDays` is matched or exceeded.

The _Details_ link behind the `renovate/stability-days` CI check points to the docs homepage.

## Screenshot

The red arrow in the screenshot points to where you need to click/hover to see the current link:

![renovate-stability-days-ci-link](https://user-images.githubusercontent.com/34918129/161601971-dddf6c5a-3987-4fe7-a6c0-e393c8ec0f78.png)

## Steps to reproduce

1. Go to https://github.com/renovatebot/renovatebot.github.io/pull/153
1. Scroll to the bottom of the page
1. Click on the green check-mark before the commit hash `d80fc15`
1. A new modal appears with "All checks have passed" as the heading
1. Look at the third list item in the modal that starts with **renovate/stability-days** and click on the Details link
1. End up at docs homepage

## Idea

How about we link to the [`stabilityDays` documentation](https://docs.renovatebot.com/configuration-options/#stabilitydays) instead? That way people will know right away what the check is for, and what Renovate config option it uses.

## Warnings

We'll need to update the link if we rename `stabilityDays` to something else like `minimumReleaseAge`: https://github.com/renovatebot/renovate/issues/12914.

**Comment by HonkingGoose** [comment]
I can still reproduce this today. I see `@viceice` give this idea a thumb-up, but I never created an issue for this idea. I think that's because I never got a clear response if we want an issue for this idea. 😄 

@rarkins do you want me to create an issue to make it so the `details` link points to our `stablityDays` config option docs instead of just the docs landing page?

**Reply by viceice** [reply]
@setchy had opened one already 😁

**Reply by setchy** [reply]
jinx @HonkingGoose  😄 - great minds think alike
- https://github.com/renovatebot/renovate/issues/21013

**Reply by HonkingGoose** [reply]
Haha! 🙂
